lib/galaxy/tools/deps/conda_util.py

```diff
@@ -0,0 +1,305 @@
+import functools
+import hashlib
+import os.path
+import re
+import shutil
+from sys import platform as _platform
+import tempfile
+
+import six
+import yaml
+
+from ..deps import commands
+
+# Not sure there are security concerns, lets just fail fast if we are going
+# break shell commands we are building.
+SHELL_UNSAFE_PATTERN = re.compile(r"[\s\"']")
+
+IS_OS_X = _platform == "darwin"
+
+# BSD 3-clause
+CONDA_LICENSE = "http://docs.continuum.io/anaconda/eula"
+
+
+def conda_link():
+    if IS_OS_X:
+        url = "https://repo.continuum.io/miniconda/Miniconda-latest-MacOSX-x86_64.sh"
+    else:
+        url = "https://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh"
+    return url
+
+
+def find_conda_prefix(conda_prefix=None):
+    """ If supplied conda_prefix is not set, default to the default location
+    for Miniconda installs.
+    """
+    if conda_prefix is None:
+        return os.path.join(os.path.expanduser("~"), "miniconda2")
```

Comment:
So I made the change in planemo, it will end up in Galaxy on the next sync after some more testing. Is this okay, or is it really important it be part of this PR. https://github.com/galaxyproject/planemo/commit/217eb9a5075bfa8a4127b6b8caff0556d4cbd516